v22.1/restore.md

```diff
@@ -20,7 +20,7 @@ You can restore:
 ## Considerations
 
 - `RESTORE` cannot restore backups made by newer versions of CockroachDB.
-- `RESTORE` is a blocking statement. To run a restore job asynchronously, use the [`DETACHED`](#detached)` option. See [Options](#options) for more usage detail.
+- `RESTORE` is a blocking statement. To run a restore job asynchronously, use the [`DETACHED`](#detached)` option. See Options for more usage detail.
```

Comment:
```suggestion
- `RESTORE` is a blocking statement. To run a restore job asynchronously, use the [`DETACHED`](#detached) option.
```

Reply:
Hey Ashley, since you have the link to detached, let's remove the end sentence as it's a little meaningless without the link. (Also removed here the typo on the backtick.)

You can commit these two suggestions in 22.1 and 22.2 from the GitHub UI before merging